### PR TITLE
Protractor 4

### PR DESCRIPTION
**Pull request by Droogans**
Includes some bugfixes too.

### CODE REVIEW

**Comment by Droogans** [comment]
There's a bug here. Every test report is saying there's image diffs, but there's nothing to commit.

This is not correct.

File: bin/systemDepsInstaller.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ You do not have one of the following supported installers available: ${Object.ke
 Please refer to https://github.com/Automattic/node-canvas#installation for more information on setting up dependencies
 ` };
 
-let needsSudoError = (installerName, installer) => { `
+let needsSudoError = (installerName, installer) => { return `
```
Comment (parlarjb) [inline bin/systemDepsInstaller.js]: I would have assumed that `return` would be implicit for the final statement in a function!

Comment (Droogans) [inline bin/systemDepsInstaller.js]: Same here. But I'm pretty sure that isn't the case as soon as a line break enters the picture. Which I didn't know at the time.

**Comment by parlarjb** [comment]
LGTM

**Comment by Droogans** [comment]
I'm going to skip Jenkins since it feels like crashing in the middle of the test for unrelated reasons.

**Pull request merged**